### ISSUE TITLE
Error message about need to run elevated on Windows is incorrect

### ISSUE THREAD

**Issue by jazzdelightsme** [moby/moby#37963]
If Docker for Windows is not running, and you use the `docker` CLI, you get an error message similar to this:

> error during connect: Get http://%2F%2F.%2Fpipe%2Fdocker_engine/v1.38/containers/json?all=1: open //./pipe/docker_engine: The system cannot find the file specified. In the default daemon configuration on Windows, the docker client must be run elevated to connect. This error may also indicate that the docker daemon is not running.

**Steps to reproduce the issue:**
1. If Docker for Windows is running, stop it.
2. Run any docker CLI command (such as "docker ps")
3. Observe error message

**Describe the results you received:**
The error teaches new users that the Docker CLI must be run elevated.

**Describe the results you expected:**
The Docker CLI does not need to run elevated, and I do not want to encourage our developers to run their development environments elevated unnecessarily.

**Output of `docker version`:**

```
Client:
 Version:           18.06.1-ce
 API version:       1.38
 Go version:        go1.10.3
 Git commit:        e68fc7a
 Built:             Tue Aug 21 17:21:34 2018
 OS/Arch:           windows/amd64
 Experimental:      false
```

The error message in question is here: [moby/moby/client/request.go#L179](https://github.com/moby/moby/blob/eff6e8bcbb212d10add7276eb1a94e4704974eb3/client/request.go#L179). User accounts should only need to be a member of the `docker-users` group, as described [in this comment](https://github.com/docker/for-win/issues/1732#issuecomment-376760330) (which is the default for the user that installed Docker).

**Comment by jhowardmsft** [comment]
It’s not strictly wrong as it’s prefixed in the default daemon configuration. Which is accurate. Feel free to suggest alternate text. IIRC we had this discussion when the message was first altered and ruled it out. From experience, I know a good 90% of users, devs especially 😇, don’t read messages accurately, and it becomes even more problematic as verbosity increases. 

An alternate might be for the D4W installer to automatically add the user installing D4W to that group. That however is not something I can do - it’s not in an OSS repo.

**Comment by jhowardmsft** [comment]
> That is already the behavior of the D4W installer.

Ah, OK. But if that's the case, why are users even hitting this error?

But anyway, I don't think we can use your suggested text, and that group isn't created in the moby repo. And in theory it can be any group name (think localisation too). 

Creation of the group and it's name is wrapped much high up than moby/moby, in this case in the D4W installer, I assume. Also, the group name may be completely different (or non-existent) on server/EE to CE used by D4W. Hence I think the right text is there currently.

**Comment by jazzdelightsme** [comment]
> why are users even hitting this error?

When the daemon is not running. :D

I tried removing myself from the `docker-users` group to see what the experience was like. In that case, I could not even launch D4W, and it told me explicitly why (because I was not a member of the group).

So then I added myself back to the group so that I could start the daemon, and then launched a single cmd window running as a different user which was *not* a member of the group. When that user tries to run a docker command, the error is:

> error during connect: Get http://%2F%2F.%2Fpipe%2Fdocker_engine/v1.38/containers/json: open //./pipe/docker_engine: **Access is denied**. In the default daemon configuration on Windows, the docker client must be run elevated to connect. This error may also indicate that the docker daemon is not running.

Given that "access denied" is already distinct from "cannot find the file specified", I wonder if we really need the sentence about being elevated to connect at all.
